Kconfig

```diff
@@ -931,6 +931,26 @@ menu "LVGL configuration"
     endmenu
 
     menu "Widgets"
+        menu "User Data"
+            config LV_USE_USER_DATA
```

Comment:
Did you also remove this from the "Feature usage" menu? (it exists there in master)

Reply:
Nope, didn't realize was duplicated. Should I just remove it from the "Feature menu", or move there the additional options I created?

Reply:
Lets keep it in the features menu, what do you think?

Reply:
I agree keep it in "Features menu" as its a field added to non widget items too.

Reply:
Done!